templates/test/live.html.ep

```diff
@@ -46,3 +46,7 @@
 </div>
 <input type="checkbox" id="scrolldown" checked="checked" />
 <label for="scrolldown">Autoscroll log</label>
+<div class="h5">Serial Terminal</div>
```

Comment:
That makes things quite a lot simpler, I can try just interleaving both serial log files. It might look a mess in a very small number of cases (e.g. the OOM killer tests cause a lot of kernel log messages to be written to ttyS0/1 as well as the hvc0), but one can inspect the individual logs after the test finishes for a precise idea of what happened.